client/package.json

```diff
@@ -95,6 +95,7 @@
     "history": "^4.9.0",
     "http-proxy-middleware": "^0.19.1",
     "jspdf": "^2.3.1",
+    "jspdf-autotable": "^3.5.23",
```

Comment:
While `jsPDF` has table support built-in, it's limited. `jspdf-autotable` is a widely used extension. (It has [no dependencies](https://github.com/simonbengtsson/jsPDF-AutoTable/blob/2022d098b27c74d666f13a96fa08b99b35658413/package.json#L68), so no need to worry about running into a CSP snag again 😅)